yagb-core/src/emulator/ppu/ppu-cgb.ts

```diff
@@ -356,4 +410,10 @@ export class PpuCgb extends PpuBase {
     private vram16Banks!: Array<Uint16Array>;
 
     private bank = 0;
+
+    private bgpi = 0;
+    private obpi = 0;
+
+    // color ram: first 64 bytes for background and second 64 bytes for objects
+    private cram = new Uint8Array(0x80);
```

Comment:
I'd prefer to split this into two arrays. We don't gain anything from this, just a few additional arithmetic operations during access.